netbsd/NetBSDProcess.h

```diff
@@ -3,7 +3,7 @@
 /*
 htop - NetBSDProcess.h
 (C) 2015 Hisham H. Muhammad
-(C) 2015 Michael McConville
+(C) 2021 Santhosh Raju
```

Comment:
Can you explain this change of the copyright header?

Reply:
That was the author of the OpenBSD's `OpenBSDProcess.h`, do you want me to retain the original author of the file?

Reply:
I'd prefer that. Also due to the many addtions you may also add the `htop dev team`.

Reply:
Sure, I shall update the headers.

Reply:
This should be done in ababf2e